### PR TITLE
chore(starters): add gatsby-bootstrap-wordpress-starter

### PR DESCRIPTION
**Pull request by billyjacoby**
## Description

Added a new starter that includes react-bootstrap and wordpress sourcing

### CODE REVIEW

File: docs/starters.yml [hunk 1]
```diff
@@ -4048,6 +4048,19 @@
...
   features:
+    - Demonstrates both query and mutation from the linked Hasura project.
+    - Uses apollo react hooks for query and mutation.
+- url: https://billyjacoby.github.io/gatsby-bootstrap-wordpress-starter/
+  repo: https://github.com/billyjacoby/gatsby-bootstrap-wordpress-starter
+  description: GatsbyJS starter with react-bootstrap, SASS, and Wordpress Sourcing
+  tags:
+    - Styling:Bootstrap
+    - Styling:SCSS
+    - Sourcing:Wordpress
```
Comment (wardpeet) [inline docs/starters.yml]: please use
```suggestion
    - CMS:Wordpress
```

**Comment by wardpeet** [comment]
@billyjacoby It also looks like you messed up merging, can you add your starter to the bottom of the file.

**Comment by wardpeet** [comment]
Friendly ping, could you fix this issue? Might be better to close this PR and start over?

**Comment by billyjacoby** [comment]
Sorry, gonna close this and reopen for adding the starter...

**Pull request closed without merging**